include/cose.h

```diff
@@ -314,14 +314,25 @@ bool COSE_Signer_SetExternal(HCOSE_SIGNER hcose, const byte * pbExternalData, si
  *  Sign routines
  */
 
+#if USE_MBED_TLS
+	typedef struct mbedtls_ecp_keypair eckey_t;
+#else
+	typedef struct eckey_t {
+		struct ec_key_st *key;
+		int group;
+	} eckey_t;
+#endif // USE_MBED_TLS
```

Comment:
Need #includes to get struct mbedtls_ecp_keypair of struct ec_key_st defined.

Reply:
Since we are using `struct mbedtls_ecp_keypair` and `struct ec_key_st` by just pointers, they are not necessary be defined, but only need to be declared (forward declaration). by writing `typedef struct mbedtls_ecp_keypair eckey_t;`, the `mbedtls_ecp_keypair` structure got itself declared.